RoboFile.php

```diff
@@ -30,6 +31,7 @@ class RoboFile extends Tasks {
    */
   public function __construct() {
     $this->started = new DateTime();
+    require_once 'vendor/autoload.php';
```

Comment:
This seems like it shouldn't be required, but the GitPHP library doesn't seem to be autoloaded otherwise without explicitly including the `autoload.php`.